### PR TITLE
Update pwd, run script

### PR DESCRIPTION
**Pull request by muralibasani**
- Update superadmin password
- run script option is added

### CODE REVIEW

File: docs/HowTo/installation/run-docker.md [hunk 1]
```diff
@@ -138,7 +138,11 @@ docker ps
 
 ### 4. Access the web interface
 
-Access the Klaw web interface using this URL: `http://localhost:9097/`
+Access the Klaw web interface using this URL:
+
+```copy-button
```
Comment (aindriu-aiven) [inline docs/HowTo/installation/run-docker.md:143]: This is cool I need to remember this copy-button

File: docs/HowTo/installation/run-source.md [hunk 2]
```diff
@@ -52,24 +52,41 @@ the source.
...
+
+   ```copy-button
+   http://localhost:9097/
+   ```
+
+7. Log in as the admin user to get started.
 
    - **Username:** `superadmin`
-   - **Password:** `kwsuperadmin123$$`
+   - **Password:** `welcometoklaw`
```
Comment (aindriu-aiven) [inline docs/HowTo/installation/run-source.md:90]: Here just mark the PW change as from version 2.6.0

File: docs/HowTo/installation/run-source.md [hunk 3]
```diff
@@ -52,24 +52,42 @@ the source.
...
+     To pass additional configuration, pass an additional parameter like
 
-8. Log in as the admin user to get started.
+     `--spring.config.location=target/classes/application.properties`
+
+     Start the web UI by running its JAR file:
+
+     `java -jar klaw-<version>.jar`
+
+     To pass additional configuration, pass an additional parameter like
```
Comment (harshini-rangaswamy) [inline docs/HowTo/installation/run-source.md:76]:
```suggestion
     To pass additional configurations, include them as follows:
```

File: docs/HowTo/installation/run-docker.md [hunk 4]
```diff
@@ -148,7 +152,8 @@ To verify the installation and access Klaw, follow the steps below:
    account:
 
    - **Username:** `superadmin`
-   - **Password:** `kwsuperadmin123$$`
+   - **Password:** `kwsuperadmin123$$` for 2.5.0 and older Klaw release
+   - **Password:** `welcometoklaw` as of 2.6.0 Klaw release
```
Comment (harshini-rangaswamy) [inline docs/HowTo/installation/run-docker.md:156]:
```suggestion
   - **Password:** `welcometoklaw` (starting from the Klaw 2.6.0 release)
```

File: docs/HowTo/installation/run-docker.md [hunk 5]
```diff
@@ -148,7 +152,8 @@ To verify the installation and access Klaw, follow the steps below:
    account:
 
    - **Username:** `superadmin`
-   - **Password:** `kwsuperadmin123$$`
+   - **Password:** `kwsuperadmin123$$` for 2.5.0 and older Klaw release
```
Comment (harshini-rangaswamy) [inline docs/HowTo/installation/run-docker.md:155]:
```suggestion
   - **Password:** `kwsuperadmin123$$` (for  Klaw releases 2.5.0 and older)
```

File: docs/HowTo/installation/run-source.md [hunk 6]
```diff
@@ -52,24 +52,42 @@ the source.
...
+
+   ```copy-button
+   http://localhost:9097/
+   ```
+
+7. Log in as the admin user to get started.
 
    - **Username:** `superadmin`
-   - **Password:** `kwsuperadmin123$$`
+   - **Password:** `kwsuperadmin123$$` for 2.5.0 and older Klaw release
```
Comment (harshini-rangaswamy) [inline docs/HowTo/installation/run-source.md:89]:
```suggestion
   - **Password:** `kwsuperadmin123$$` (for  Klaw releases 2.5.0 and older)
```

File: docs/HowTo/installation/run-source.md [hunk 7]
```diff
@@ -52,24 +52,42 @@ the source.
...
+   - **Password:** `welcometoklaw` as of 2.6.0 Klaw release
```
Comment (harshini-rangaswamy) [inline docs/HowTo/installation/run-source.md:90]:
```suggestion
   - **Password:** `welcometoklaw` (starting from the Klaw 2.6.0 release)

**Review by harshini-rangaswamy — CHANGES_REQUESTED** [review]
Minor changes to the text

**Review by harshini-rangaswamy — APPROVED** [review]

**Review by aindriu-aiven — APPROVED** [review]
LGTM

**Pull request merged**